package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "PORT=8080 next dev",
     "build": "next build",
+    "debug:standalone": "next build && cp -rf public .next/standalone/ && cp -rf .next/static .next/standalone/.next/static && cp -rf node_modules/web-worker/cjs/browser.js .next/standalone/node_modules/web-worker/cjs/ && node .next/standalone/server.js",
```

Comment:
This should start the server using the same port as `dev` (8080), for consistency with cors headers.